src/challenge/Students.java

```diff
@@ -0,0 +1,24 @@
+package challenge;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Students {
+    public static void main(String[] args) {
+        ArrayList student = new ArrayList();
+        student.add("Maria");
+        student.add("Ion");
+        student.add("George");
+        student.add("Andreea");
+        student.add("Bianca");
+        student.add("Cristina");
+        student.add("Tudor");
+        student.add("Alin");
+        student.add("Stefan");
+        student.add("Denisa");
+        ArrayList graduateStudent = new ArrayList();
+        graduateStudent.addAll(student);
```

Comment:
We can use a loop to add student list to graduate student list?

Reply:
What loop? can I use for?

Reply:
Yes, a for loop would be great.